samples/javascript_nodejs/52.logger-bot/resources/echo.chat

```diff
@@ -0,0 +1,7 @@
+user=Vishwac
+bot=EchoBot
+
+user: Hi
+bot: 1: You said "Hi"
+user: Hello
+bot: 2: You said "Hello"
```

Comment:
Not relevant to this specific file - but few additional comments - 
* We should have this numbered 25. instead of 52.
* Add root readme.md update to include link to this sample

Reply:
Changed the number and added a readme link